SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -1,72 +1,90 @@
 
-* **Stop**</br>
-Stops `spatial local launch`.
+## SpatialOS GDK for Unreal toolbar
 
-* **Inspector**</br>
-Opens the [Inspector](https://docs.improbable.io/reference/latest/shared/glossary#inspector) (SpatialOS documentation) in a browser.
+The GDK toolbar provides several functions required for building and launching your client and server-workers from inside the Unreal Editor.
 
-## Settings
+### Add the GDK toolbar to your Unreal project
 
-The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal GDK** > **Toolbar**.
+> Note: If you based your project off the [StarterProject](https://github.com/spatialos/UnrealGDKStarterProject), the toolbar is already enabled.
 
- ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-settings.png)
+To enable the GDK toolbar, navigate to **Edit** > **Plugins** inside the Unreal Editor and scroll down to the bottom. Select the **SpatialOS Unreal GDK** section and enable the toolbar:
 
-* **Configuration**
+![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/enable-toolbar.png)
+
+#### Buttons
+
+The GDK toolbar has five features mapped to individual buttons and is displayed in the main editor toolbar to the right of the `Launch` button:
+
+ ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-buttons.png)
 
-    * **Project root folder**</br>
-    If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`.
+You can also access these from the **Window** menu:
+
+ ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/window-access.png)
 
-    * **Launch configuration**</br>
-    The [launch configuration file](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config) (SpatialOS documentation) to use when running `spatial local launch` using the `Launch` button.
+| Button | Description |
+| --- | --- |
+| Snapshot | Generates a [SpatialOS snapshot (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#snapshot). | 
+| Schema | Creates `schema` for your Unreal project. |
+| Launch | Runs [`spatial worker build build-config` (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-worker-build-build-config) to build worker configs and runs `spatial local launch` with the launch configuration specified in the settings (see [below](#settings)). |
+| Stop | Stops `spatial local launch`. |
+| Inspector | Opens the [Inspector (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#inspector) in a browser. |
```

Comment:
Didn't realise the Inspector was also in our GDK glossary - remove `(SpatialOS documentation) and change the link to `({{urlRoot}}/content/glossary#inspector)`

Reply:
Keep the padded cell!!! :-)